base/remote/R/remote.copy.from.R

```diff
@@ -45,5 +45,37 @@ remote.copy.from <- function(host, src, dst, options = NULL, delete = FALSE, std
     }
   }
   PEcAn.logger::logger.debug("rsync", shQuote(args))
-  system2("rsync", shQuote(args), stdout = TRUE, stderr = as.logical(stderr))
+  out <-
+    system2("rsync", args, stdout = "", stderr = as.logical(stderr))
+  
+  # Informative errors from rsync man page
```

Comment:
I appreciate the desire for clarity, but I'd rather send people to the rsync docs for these error messages. We use whatever rsync version is installed on the host system, so hard-coding values here feels like an invitation for them to get out of sync. 

I assume rsync doesn't often change the meaning of exit codes between versions, but if they add a new code that's not on this list then this switch will return an empty string instead of the numeric value.

Reply:
Yeah, this makes sense.  I'm going to keep errors (anything other than 0) as `logger.severe()` though.